Docker/Alfresco/ark-haproxy/start_haproxy.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -eu -o pipefail
+
+echo "Building certificate bundle for HAProxy"
+mkdir -p /app/cert
+chown root:root /app/cert
+chmod 700 /app/cert
+mycrt="/etc/minipki/issued/ark-haproxy.crt"
+mykey="/etc/minipki/private/ark-haproxy.key"
+cacrt=/etc/minipki/ca.crt
+cat "$mycrt" "$cacrt" "$mykey" > /app/cert/bundle.pem
+chmod 600 /app/cert/bundle.pem
+
+echo "Creating an X.509 self-signed certificate"
+mkdir /etc/certs /etc/keys
+chmod 755 /etc/certs
+chmod 700 /etc/keys
+openssl req -x509 -nodes -days 9000 -newkey rsa:2048 -keyout /etc/keys/key.pem -out /etc/certs/cert.pem -subj "/CN=haproxy"
+chmod 700 /etc/keys/key.pem
+
+cd /app
+cat haproxy.cfg.tmpl \
```

Comment:
Eventually could we allow the user to specify their own pre-existing valid certificate, maybe also alternative certificate generation strategies?